cpp/data/installer/ixtheo.cronjobs

```diff
@@ -12,6 +12,7 @@ LOG_DIR=/usr/local/var/log/tuefind
 0 0 * * * "$BIN/log_rotate" --max-rotations=4 "$LOG_DIR" "(?<!(java_mem_stats))\\.log$"
 0 0 * * * "$BIN/log_rotate" --no-of-lines-to-keep=200 "$LOG_DIR" "^java_mem_stats\\.log$"
 30 0 * * * cd "$BSZ_DATEN" && "$BIN/purge_old_data.py" "$EMAIL" > "$LOG_DIR/purge_old_data.log" 2>&1
+30 0 * * * "$BIN/generate_new_journal_alert_stats.sh"
```

Comment:
- We need a system_update script as well to add this cronjob to all existing servers
(- Also we could register ixtheo / relbib as different cases, see other comment)

Reply:
Done.